packages/main/tests/resources/secrets.yaml

```diff
@@ -0,0 +1,17 @@
+credentials:
+    google:
+        apikey: '1234567890'
```

Comment:
This is a string, exactly as in the JSON secrets file. If you want a number, just remove the quotes, like you'll be doing in the JSON format too. They're symmetrical.